src/components/Button/Button.js

```diff
@@ -14,7 +14,7 @@ const Button = styled.button`
   line-height: ${props => props.height};
 
   @media (min-width: ${props => props.theme.breakpoints.laptop}) {
-    font-family: "Raleway";
+    font-family: 'Raleway';
```

Comment:
It seems like because this font family is inside a media query, it's only Raleway on a smaller screen size.